generator/internal/rust/annotate.go

```diff
@@ -61,6 +61,9 @@ type serviceAnnotations struct {
 	// in `PascalCase`. Notably, names like `IAM` *must* become `Iam`, but
 	// `IAMService` can stay unchanged.
 	Name string
+	// The source specification package name mapped to Rust modules. That is,
+	// `google.service.v1` becomes `google::service::v2`.
```

Comment:
I would hope `v1` gets mapped to `v1`, not `v2`.

:)

Reply:
Fixed.